.github/workflows/lint.yml

```diff
@@ -17,20 +17,20 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 6
     steps:
-      - uses: actions/setup-go@v4.0.0
+      - uses: actions/setup-go@v4.0.1
         with:
-          go-version: 1.19
```

Comment:
Are there any plans to update the golang version of ramus as well?

Reply:
Yes. Latest ostracon requires 1.20 so all depending on it needs to be updated.